modules/v1/users/models/query/InvitationQuery.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace app\modules\v1\users\models\query;
+
+use app\modules\v1\users\models\Invitation;
+use yii\db\ActiveQuery;
+
+/**
+ * This is the ActiveQuery class for [[\app\modules\v1\users\models\Invitation]].
+ *
+ * @see \app\modules\v1\users\models\Invitation
+ */
+class InvitationQuery extends ActiveQuery
+{
+    /**
+     * {@inheritdoc}
+     * @return Invitation[]|array
+     */
+    public function all($db = null)
+    {
+        return parent::all($db);
+    }
+
+    /**
+     * {@inheritdoc}
+     * @return Invitation|array|null
+     */
+    public function one($db = null)
+    {
+        return parent::one($db);
+    }
+
+    /**
+     * Return not used invitation link
+     *
+     * @return mixed
+     */
+    public function notUsed()
+    {
+        return $this->andWhere(['use_date' => null]);
```

Comment:
@skavt Use table name as well when comparing this field, ex: `andWhere([Invitation::tableName().'.someField' => null])`